### ISSUE TITLE
root is partially ignored for creating packages

### ISSUE THREAD

**Issue by halkeye** [Microsoft/tfs-cli#62]
I'm creating all the files I want to package up in the dist directory.

```
$ ./node_modules/.bin/tfx extension create --root dist --output-path Packages
$ cd Packages
$ unzip saucelabs-beta.saucelabs-tasks-0.2.0.vsix
$ md5 ../dist/sod-build-info/scripts/dump.js sod-build-info/scripts/dump.js ../sod-build-info/scripts/dump.js
MD5 (../dist/sod-build-info/scripts/dump.js) = 9b1880b43f26d4fff8d7f7de6b0e5da5
MD5 (sod-build-info/scripts/dump.js) = 6e25b092202ef26e730da592c9c76af2
MD5 (../sod-build-info/scripts/dump.js) = 6e25b092202ef26e730da592c9c76af2
```

as you can see from the md5 (and i've checked the file contents) its pulling the one from cwd not root.
I don't have any other files my build process is yet messing with (other than vss-extension but its not in the zip).

Is this the right use of --root? Is it a bug?

**Comment by halkeye** [comment]
☁  vso-sauce-ondemand-plugin [develop] npm ls tfx-cli
vso-sauce-ondemand-plugin@0.2.0 /Users/gavinmogan/git/vso-sauce-ondemand-plugin
└── tfx-cli@0.3.7

**Comment by trevorsg** [comment]
Looks like a bug. I'll investigate. Thanks!

**Comment by trevorsg** [comment]
Fixed in c42475b. Npm has been updated. Thanks for reporting.

**Comment by halkeye** [comment]
yay! b2ad002ed4c53f8a1e28647e0282f25e4f3327d8 fixes it
